BridgeAppSDK/SBASurveyFactory.swift

```diff
@@ -68,6 +68,12 @@ open class SBASurveyFactory : SBABaseSurveyFactory {
         return json["steps"] as? [NSDictionary]
     }()
     
+    // This is a time-consuming operation (b/c data groups have moved to the keychain)
+    // so use a lazy load to only load when called, but then retain the result in memory.
+    lazy open var currentDataGroups: Set<String> = {
```

Comment:
You shouldn't need to access it directly? You can access its properties through the profile manager, and they're all set up already as ProfileItems in either BridgeAppSDK (for the base properties) or the app (for the custom attributes). The one for data groups has the profile key "participantDataGroups".